Add handling for invalid pages during autoExport (#7574)
zeit/next.js: errors/page-without-valid-component.md

# Page Without Valid React Component

#### Why This Error Occurred

> While auto exporting a page a valid React Component wasn't found. This could mean you have a file in `pages` that does `export default { some: 'object' }` instead of a React Component.

Reviewer: ```suggestion
While auto exporting a page a valid React Component wasn't found. This could mean you have a file in `pages` that exports something that is not a React Component.
```
